db/schema.rb

```diff
@@ -270,6 +279,12 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "admin", default: false, null: false
+    t.string "short_name"
+    t.string "facebook_user_id"
+    t.string "gr_master_person_id"
+    t.string "google_user_id"
+    t.string "apple_user_id"
```

Comment:
Now that there are three *_user_id, might there eventually be four? Would it be better to have a single user_id field and a type field?

Reply:
I'd like to reserve the possibility to have a user linked to multiple identity providers in the future. Consolidating this into a `user_id` and `type` field would prevent that from happening without future refactoring

Reply:
👍